help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,158 @@
+
+### Destination attributes {#attributes}
+
+In the [[!UICONTROL Select attributes]](../../ui/activate-streaming-profile-destinations.md#select-attributes) step, Adobe recommends that you select a unique identifier from your [union schema](../../../profile/home.md#profile-fragments-and-union-schemas). Select the unique identifier and any other XDM fields that you want to export to the destination.
+
+### Mapping example: activating profile updates in [!DNL Pega Customer Decision Hub] 
+
+Below is an example of correct identity mapping when activating profile updates in [!DNL Pega Customer Decision Hub].
+
+Selecting source fields:
+
+* Select an Identifier (For eg: CustomerID) as source identity that uniquely identifies a profile in Adobe Experience Platform and  Pega Customer Decision Hub.
+* Select XDM source profile attribute changes that need to be updated in Pega Customer Decision Hub.
+
+Selecting target fields:
+
+* Select the `CustomerID` namespace as target identity.
+* Select destination profile attribute names that need to be mapped to corresponding XDM source profile attributes.
+
+![Identity mapping](../../assets/catalog/http/pega/pega-source-destination-mapping.png)
+
+## Exported data / Validate data export {#exported-data}
+Your exported [!DNL Experience Platform] data lands in your [!DNL HTTP] destination in JSON format. For example, the export snippet below contains a profile that is qualified for a certain segment, is a member of another two segments, and exited another segment. The export also includes the profile attribute first name (*FirstName*), last name (*LastName*), date of birth (*BirthDate*), and Customer Identfier (*CustomerID*) for this profile.
+
+```json
+{
+  "CustomerID": "CUSTOMER-1021",
+  "Attributes": {
+    "BirthDate": "1975-08-29",
+    "FirstName": "John",
+    "LastName":"Doe"
+  },
+  "Segments": [{
+    "SegmentID": "04a81716-43d6-4e7a-a49c-f1d8b3129ba91",
+    "Name": "Interested in iPhone 13",
+    "LastQualificationTime": "2020-05-25T21:24:39Z",
+    "Status": "existing",
+    "Version": "15",
+    "ValidUntil": "2020-06-25T21:24:39Z",
+    "Namespace": "AAM"
+    }, {
+    "SegmentID": "53cba6b2-a23b-454a-8069-fc41308f1c0f",
+    "Name": "Interested in Unlimited Data Plan",
+    "LastQualificationTime": "2020-05-25T23:37:33Z",
+    "Status": "exited",
+    "Version": "3",
+    "ValidUntil": "2020-07-25T23:37:33Z",
+    "Namespace": "AAM"
+    }]
+ }
+
+```
+For [!DNL Pega], a successful Segment membership update for a profile would insert the segment identifier, name and statuses in Pega Marketing segment membership datastore. The segment membership data can be assoicated in Pega Customer Profile designer that can be used in Engagement polices for next-best-action decisioning and/or predictors in Adaptive models. 
+
+![exported-data](../../assets/catalog/http/pega/pega-profile-designer-associate.png)
```

Comment:
Two comments: 

1. Can we please insert some context in between the images, to exactly detail what someone should be looking for in each image? 
2. instead of `exported-data`, can we please also expand the alt-text for each image to describe what it is that we are seeing? A good example is the alt-text for the image on row 75, here: https://github.com/AdobeDocs/experience-platform.en/pull/86/files#diff-b1e44336c76d16e63669434c9bebe46d165d8d5cfdac02433adef165f022dbe6R75